db/migrations/000001_create_activities_table.up.sql

```diff
@@ -0,0 +1,25 @@
+BEGIN;
+
+CREATE TABLE activities
```

Comment:
(non-blocker): It would be nice to have comments or maybe a `docs/entities.md` explaining what this is about. The context of `fork` and `content`, to which MSs belong.

Reply:
@rluders Do you have suggestions to explain `content_uuid`?

```
Main content of the activity. This content has a specific type (video, pdf, questionnaire, slide, etc.), each type of content has its own Micro Service.
```

Reply:
@marianysilva I think that this sounds correct. I wonder if it would not get confused with the "Content Microservice" that will connect with the bucket to serve as a resource/content catalog for the activities. 🤔 Maye we can rename the "Content Microservice" to "Resource Microservice" or "File Microservice" or "Bucket Microservice"... So, yeah, the description is correct.

Reply:
TY